### ISSUE TITLE
Pictures are too big

### ISSUE THREAD

**Issue by erciccione** [monero-ecosystem/outreach-docs#31]
Pictures are extremely big and make the whole repository extremely heavy. Only the portuguese section is about 103.9 MB. Pictures should be lighter, especially the ones in.tif format in `.../PSA/files`. Some are almost 10 MB and **2550x1275 px**. I don't really think they need to be that big.

**Comment by erciccione** [comment]
@xmrhaelan @lh1008

**Comment by lh1008** [comment]
OK, understand. I think we actually don't need those images once they are already inside the .sla file. Let me see what I can do. 

@thunderosa is there any way we can lighter the images?

**Comment by lh1008** [comment]
@erciccione we will have to change the image resolution. I tried removing the image from the folder but that didn't work. I will be addressing this with @thunderosa. Thank you :).

**Comment by erciccione** [comment]
The monero-outreach repository is **752.8 MB** big at the moment. For comparison: 
Monero-gui = 46.2 MB
monero-GUI-guide = 9.2 MB

**Comment by lh1008** [comment]
@erciccione hi,

We are looking into this. Leaving this message to keep you updated. I will see if I'm able to make them smaller.

**Comment by lh1008** [comment]
@erciccione found a way to reduce images. Working on it right now. Will submit a PR soon. 

Thanks.

**Comment by lh1008** [comment]
See #47.

**Comment by lh1008** [comment]
Can we close this issue and keep on working on the size, or do we keep it as a reminder?

**Comment by erciccione** [comment]
I would keep it as a reminder. This repo is still way too big

**Comment by Thunderosa** [comment]
Yeah, I noticed this. Vector images should be used and not bitmap (tif etc) and the size will return to normal. The full color version should be about 1.2MB and the printer friendly should be around .5MB.

Also make sure that the .pdf is being rendered appropriately. Press quality isn't really needed most of the time.

**Comment by Thunderosa** [comment]
High res working files are always going to be large. In the repo you could opt to keep just the tifs and not the .psd and that would save some size. Storing them in a .rar file would also save some space.

On the Quick Facts the problem is that bitmap images were used instead of vector. The pdf's should come out to be about 1.2MB for the full color and 500K-600K for the printer friendly.

**Comment by erciccione** [comment]
@Thunderosa Got it, thanks for the update. Could you PR those 2 changes? (remove the .psd files and keep the tifs + remove bitmap images and replace them with vectors)

**Comment by erciccione** [comment]
Just FYI, these as some warnings i get when pushing to the repo:

```
remote: warning: GH001: Large files detected. You may want to try Git Large File Storage - https://git-lfs.github.com.
remote: warning: See http://git.io/iEPt8g for more information.
remote: warning: File monero-outreach-docs/translations/zh-hans/PingFang.ttc is 76.30 MB; this is larger than GitHub's recommended maximum file size of 50.00 MB
remote: warning: File monero-outreach-docs/translations/zh-hans/Songti.ttc is 63.85 MB; this is larger than GitHub's recommended maximum file size of 50.00 MB
```

I don;t know why those two files are so big, but they need to be reduced.

**Comment by lh1008** [comment]
You are right @Thunderosa, actually, we have removed all the images and all the .sla files because they are too heavy to work with. We are only leaving the PDF's and .md file formats to work the translations. This is being done through the #86. 
I suggested the file edits should be done in a different platform or work them through our communication channels and then post the final documents. What do you think?

**Comment by erciccione** [comment]
> We are only leaving the PDF's and .md file formats to work the translations. This is being done through the #86.

@lh1008 Just wanted to specify that your PR #86 is also removing PDFs
